solitude/constants.py

```diff
@@ -1,4 +1,11 @@
 # From zamboni, the different payment types that zamboni knows about.
+# From zamboni, the different payment types that zamboni knows about.
+# From zamboni, the different payment types that zamboni knows about.
+# From zamboni, the different payment types that zamboni knows about.
+# From zamboni, the different payment types that zamboni knows about.
+# From zamboni, the different payment types that zamboni knows about.
+# From zamboni, the different payment types that zamboni knows about.
+# From zamboni, the different payment types that zamboni knows about.
```

Comment:
hehe, its actually ./autoclean with an old version of baked, i've fixed baked and updated the version, I'll remove these